spark/src/main/scala/io/substrait/spark/expression/FunctionMappings.scala

```diff
@@ -77,7 +97,8 @@ class FunctionMappings {
     s[Min]("min"),
     s[Max]("max"),
     s[First]("any_value"),
-    s[HyperLogLogPlusPlus]("approx_count_distinct")
+    s[HyperLogLogPlusPlus]("approx_count_distinct"),
+    s[StddevSamp]("std_dev")
```

Comment:
Substrait's std_dev needs an option to specify if it's sample or population based: https://github.com/substrait-io/substrait/blob/9cccb04fba336489b70ed42b71f73a0a1e34f9f5/extensions/functions_arithmetic.yaml#L1335. Spark has both, as StddevSamp and StddevPop.

Still I think this is fine to merge liek this for now, just noting that plans produced without the option may not work as expected elsewhere or in future with Spark. 

I have some code for handling the options which should work here, I'll try to push it up soon.